Makefile

```diff
@@ -19,7 +19,7 @@ BOX_URL="https://github.com/humbug/box/releases/download/3.10.0/box.phar"
 
 PHP_CS_FIXER=./.tools/php-cs-fixer
 PHP_CS_FIXER_URL="https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v2.18.2/php-cs-fixer.phar"
-PHP_CS_FIXER_CACHE=build/cache/.php_cs.cache
+PHP_CS_FIXER_CACHE=.php_cs.cache
```

Comment:
php-cs-fixer doesn't create folder if it doesn't exist. and I wouldn't create an additional step just for this case.